site/docs/tools/iceberg/hive.md

```diff
@@ -0,0 +1,70 @@
+
+- `iceberg.catalog.<catalog_name>.warehouse` : The location where to store Iceberg tables managed by Nessie catalog. This will be the same location that is used to create an Iceberg table as it shown below.
+
+- `iceberg.catalog.<catalog_name>.ref` : The _current_ Nessie branch. Note that Hive doesn't support the notation of `table@branch`, therefore everytime you want to execute against a specific branch, you will need to set this property to point to the working branch, e.g: `SET iceberg.catalog.<catalog_name>.ref=main`.
+
+- `iceberg.catalog.<catalog_name>.uri`: The location of the Nessie server.
+
+For example, the following properties work for catalog named `nessie`:
+
+```
+SET iceberg.catalog.nessie.warehouse=/home/user/notebooks/nessie_warehouse;
+SET iceberg.catalog.nessie.ref=dev;
+SET iceberg.catalog.nessie.catalog-impl=org.apache.iceberg.nessie.NessieCatalog;
+SET iceberg.catalog.nessie.uri=http://localhost:19120/api/v1;
+```
+
+## Create tables
+
+Whenever Hive creates an Iceberg table, it will create it as external table that is managed by Iceberg catalog (in this case Nessie Catalog), thus, some properties need to be provided in order to create a Iceberg tables in Hive:
+
+```
+CREATE TABLE database_a.table_a (
+  id bigint, name string
+) PARTITIONED BY (
+  dept string
+) STORED BY 'org.apache.iceberg.mr.hive.HiveIcebergStorageHandler'
+  LOCATION '/path_nessie_warehouse/database_a/salaries
+  TBLPROPERTIES ('iceberg.catalog'='<catalog_name>', 'write.format.default'='parquet');
+```
+Whereby the above properties are explained as below:
+- `STORED BY 'org.apache.iceberg.mr.hive.HiveIcebergStorageHandler'` : Since Hive doesn't use a custom global catalog, we tell Hive here that individual table will be managed by Iceberg's catalog through `org.apache.iceberg.mr.hive.HiveIcebergStorageHandler`.
+
+- `LOCATION '/path_nessie_warehouse/database_a/salaries` : As mentioned before, Hive will create Iceberg tables as external tables and thus, location of the data files needs to be provided. This location is the same location that is provided in `iceberg.catalog.<catalog_name>.warehouse` in addition to the database and table names.
+
+- `'iceberg.catalog'='<catalog_name>'` : The custom Iceberg catalog to be used to manage this table.
+
+- `'write.format.default'='parquet'` : The format that is used to store the data, this could be anything that is supported by Iceberg, e.g: ORC.
+
+
+## Writing and reading tables
+
+To read and write into tables that are managed by Iceberg and Nessie, typical Hive SQL queries can be used. Refer to this documentation [here](https://iceberg.apache.org/hive/#querying-with-sql) for more information.
+
+**Note**: Hive doesn't support the notation of `table@branch`, therefore everytime you want to execute against a specific branch, you will need to set this property to point to the working branch, e.g: `SET iceberg.catalog.<catalog_name>.ref=main`. E.g:
```

Comment:
tangential: Should we perhaps support something like the `branch#hash` notation for `ref` config settings to allow utilizing the full Nessie feature set in Hive (and possibly other restrictive env.)?

Reply:
I guess this can be a different discussion if we support this or not in Hive. I have a ticket for this here: https://github.com/projectnessie/nessie/issues/1985, we can discuss there the alternatives and how we can support Hive better.

Reply:
FYI: also related to the "reference string syntax party" are #1352 + #1452